commercial/app/model/commercial/events/LiveEventAgent.scala

```diff
@@ -64,10 +62,12 @@ object LiveEventAgent extends ExecutionContexts with Logging {
 
     val liveEvents =
       for{
-        eventImages <- fetchAndParseLiveEventImages
+        eventMembershipInfo <- fetchAndParseLiveEventsMembershipInfo
         ParsedFeed(events, _) <- Eventbrite.parsePagesOfEvents(feedMetaData, feedContent)
-      } yield events
-            .map ( event => LiveEvent(event, eventImages find (_.eventId == event.id)) )
+      } yield events flatMap { event =>
+        val matchingMembershipInfo = eventMembershipInfo find (_.id == event.id)
+        matchingMembershipInfo map (LiveEvent(event, _) )
```

Comment:
Could convert both lists into hashmaps and then use `merged` to combine them.  But it would possibly be less readable.